amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/field/APIField.java

```diff
@@ -32,6 +34,9 @@
 
     @JsonUnwrapped
     private APIType apiType;
+    
+    @JsonIgnore
+    private Class<?> internalFieldType;
```

Comment:
After an analysis with Viorel, we've come up with the conclusion that this field is not required. Also in APIType, elementType is no longer needed. If APIType#fieldType is anything but list then type is the class of the field. If APIType#fieldType is list, then type is the class of the elements of that list.

Reply:
Agree